onnx/onnx-operators-ml.proto

```diff
@@ -36,22 +36,6 @@ enum OperatorStatus {
 message FunctionProto {
   // The name of the function, similar usage of op_type in OperatorProto.
   optional string name = 1;
-  
```

Comment:
no, this should not be touched, Removing these fields means the function is not complete.